include/hal/common-mock-func.sh

```diff
@@ -0,0 +1,486 @@
+  fi
+
+  if [ "$_arg_f" = "/sys/class/mei/mei0/fw_status" ]; then
+  # Here we emulate MEI controller status file presence, check check_if_fused
+  # func for more inf.:
+    [ "$TEST_MEI_CONF_PRESENT" = "true" ] && return 0
+  fi
+
+  return 1
+}
+
+fsread_tool_cat_mock(){
+  local _file_to_cat
+  _file_to_cat="$1"
+
+  # Note, Test folder here comes from dmesg_i2c_hid_detect_mock, which is being
+  # called before fsread_tool_cat_mock in touchpad-info script:
+  if [ "$_file_to_cat" = "/sys/bus/i2c/devices/Test/firmware_node/hid" ] && [ -n "$TEST_TOUCHPAD_HID" ]; then
+  # Used in touchpad-info script.
+    echo "$TEST_TOUCHPAD_HID" 1>&1
+  # Note, Test folder here comes from dmesg_i2c_hid_detect_mock, which is being
+  # called before fsread_tool_cat_mock in touchpad-info script:
+  elif [ "$_file_to_cat" = "/sys/bus/i2c/devices/Test/firmware_node/path" ] && [ -n "$TEST_TOUCHPAD_PATH" ]; then
+  # Used in touchpad-info script.
+    echo "$TEST_TOUCHPAD_PATH" 1>&1
+  elif [ "$_file_to_cat" = "/sys/class/power_supply/AC/online" ] && [ "$TEST_AC_PRESENT" = "true" ]; then
+  # Emulating AC adadpter presence, used in check_if_ac func.:
+    echo "1" 1>&1
+  elif [ "$_file_to_cat" = "/sys/class/mei/mei0/fw_status" ] && [ "$TEST_MEI_CONF_PRESENT" = "true" ]; then
+  # Emulating MEI firmware status file, for more inf., check check_if_fused
+  # func.:
+    echo "smth" 1>&1
+    echo "smth" 1>&1
+    echo "smth" 1>&1
+    echo "smth" 1>&1
+    echo "smth" 1>&1
+    # Emulating Intel Secure Boot Fuse status, check check_if_fused func. for
+    # more inf. 4... if fused, and 0 if not:
+    echo "${TEST_INTEL_FUSE_STATUS}0000000" 1>&1
+    echo "smth" 1>&1
+  else
+    echo "${FUNCNAME[0]}: ${_file_to_cat}: No such file or directory"
+
+    return 1
+  fi
+
+  return 0
+}
+
+################################################################################
+# setpci
+################################################################################
+TEST_ME_OP_MODE="${TEST_ME_OP_MODE:-0}"
+
+setpci_check_me_op_mode_mock(){
+# Emulating current ME operation mode, check functions check_if_me_disabled and
+# check_me_op_mode:
+  echo "0$TEST_ME_OP_MODE" 1>&1
+
+  return 0
+}
+
+################################################################################
+# lscpu
+################################################################################
+TEST_CPU_MODEL="${TEST_CPU_MODEL:-test}"
+
+lscpu_common_mock(){
+# Emulating CPU model, check update_workflow function. The model should look
+# like i5-13409:
+  echo "12th Gen Intel(R) Core(TM) $TEST_CPU_MODEL"
+
+  return 0
+}
+################################################################################
+# rdmsr
+################################################################################
+TEST_MSR_CAN_BE_READ="${TEST_MSR_CAN_BE_READ:-true}"
+TEST_FPF_PROGRAMMED="${TEST_FPF_PROGRAMMED:-0}"
+TEST_VERIFIED_BOOT_ENABLED="${TEST_VERIFIED_BOOT_ENABLED:-0}"
+
+rdmsr_secure_boot_status_mock(){
```

Comment:
Does this have anything to do with secure boot or with boot guard? If with boot guard then maybe change name?

Reply:
Addressed [here](https://github.com/Dasharo/dts-scripts/compare/8e94cbc7f99ffe845b06f6f742c9b9863dec795e..992062280bbd17d42f45e14d98c3e682971d4ee4)